WARNING  urllib3.connectionpool:connectionpool.py:871 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10e46ab50>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/b3e1ebb441f019e1b06f2764e092d5e5/url
WARNING  urllib3.connectionpool:connectionpool.py:871 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10e46a070>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/b3e1ebb441f019e1b06f2764e092d5e5/url
WARNING  urllib3.connectionpool:connectionpool.py:871 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10e31ec10>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/b3e1ebb441f019e1b06f2764e092d5e5/url
WARNING  urllib3.connectionpool:connectionpool.py:871 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10e333880>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/b3e1ebb441f019e1b06f2764e092d5e5/window
WARNING  urllib3.connectionpool:connectionpool.py:871 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10e333d00>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/b3e1ebb441f019e1b06f2764e092d5e5/window
WARNING  urllib3.connectionpool:connectionpool.py:871 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10e333490>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/b3e1ebb441f019e1b06f2764e092d5e5/window